filecoin-proofs/src/api.rs

```diff
@@ -155,19 +156,19 @@ pub fn generate_post(
         sector_size,
         sector_count,
         challenge_seed,
-        &faults,
+        faults,
     );
 
     // Match the replicas to the challenges, as these are the only ones required.
     let challenged_replicas: Vec<_> = challenges
         .iter()
         .map(|c| {
-            if let Some(replica) = replicas.get(c.sector as usize) {
+            if let Some(replica) = replicas.get(u64::from(c.sector_id) as usize) {
```

Comment:
blocking: if we do this, `Challenge` should contain a `SectorId`

Reply:
Word. I [made that change](https://github.com/filecoin-project/rust-fil-proofs/pull/840/files#diff-7ff052526c811cb90665a7388bfdcc1fR200) already.

Reply:
Just changed the field name back to `sector`, from `sector_id` as per your request.